src/components/organisms/EarthEditor/CanvasArea/hooks.ts

```diff
@@ -140,6 +149,39 @@ export default (isBuilt?: boolean) => {
     document.title = title;
   }, [isBuilt, title]);
 
+  const [updateLayerLatLng] = useUpdatePropertyValueLatLngMutation();
+
+  const handleDragLayer = useCallback((_layerId: string, _position: LatLngHeight | undefined) => {
```

Comment:
_layerId and _position aren't being used in this function. All this does is set dragging to true.

Reply:
I think the interface of the component which are passed this function as props shouldn't change because of this implementation. The cesium-dnd expect to recieve this type of function and to keep the interface stable, I chose this way.
I assume `_` is conventionally used for ignored params.
@rot1024 How do you think about this?